client/layout/store-notices/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import PropTypes from 'prop-types';
+import { Notice } from '@wordpress/components';
+
+/**
+ * WooCommerce dependencies
+ */
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import withSelect from 'wc-api/with-select';
```

Comment:
Since the options store is already on `@wordpress/data` seems like we should be able to use `import { withSelect } from '@wordpress/data';` instead as an external dependency?

Reply:
Good point! I modified it in the commit 96194bc